src/core/algorithms.rs

```diff
@@ -0,0 +1,230 @@
+//! Miscelanious algorithms for grids.
+
+use std::fmt::Display;
+
+use ndarray::Array2;
+
+/// Error for flood_fill
+#[derive(Debug)]
+pub enum FFError {
+    /// Denotes an invalid seed point was provided.
+    InvalidSeed,
+}
+
+impl Display for FFError {
```

Comment:
Do we know why (according to codecov) this doesn't get exercised during the test that generates an error?

Reply:
Because the value is not printed, the error is just thrown. I'm trying to fix it so codecov behaves.

Reply:
Gotcha. I assume we can do something like test the error message?